modes/micromobility.md

```diff
@@ -37,9 +45,24 @@ The `trip_attributes` array is not used with the micromobility mode.
 The `vehicle_attributes` array may have the following key value pairs:
 
 - `year` (integer)
-- `manufacturer` (string)
+- `make` (string)
 - `model` (string)
 
+### Propulsion Types
+
+#### Valid for vehicle_types: bicycle, scooter, moped, other
+
+| `propulsion`      | Description                                            |
```

Comment:
Shouldn't we just list the values possible here, and not re-iterate the Description info, since that will just be shared in one place for all modes?

Reply:
Fixed in 5e23f0b